### ISSUE TITLE
Opsctl integration

### ISSUE THREAD

**Issue by cornelius-keller** [giantswarm/roadmap#643]
### User Story:
As a giant swarm employee on call I want `opsctl` support CAPO clusters, so that I don't need any special knowledge to give support. 

### Tasks
- [x] list installations
- [x] show installations
- [x] #703
- [x] kgs login
- [x] secret encryption / decryption #591
- [x] kubeconfig
- [x] open

**Comment by cornelius-keller** [comment]
@cornelius-keller  create sub issues for each integration

**Comment by yulianedyalkova** [comment]
Afaik `opsctl list installations` and `opsctl show installation` already work for CAPO clusters.

**Comment by cornelius-keller** [comment]
Notes from planning:
- support creating kubeconfig by using ca + private key from the MC
- see if we can use kubectl gs login like on ginger 
 ( `kubectl gs login --cluster-admin=true --certificate-group "system:masters" --certificate-ttl 24h  --workload-cluster int01 ginger` works )

**Comment by pipo02mix** [comment]
`opsctl open` command still does not work, it misses the installation name

```~ » opsctl open -a happa -i $INSTALLATION --no-browser                                                                                                                                  (flux01-admin@flux01/default)
https://happa.g8s.test.gigantic.io:443/admin-login
-----------------------
~ » opsctl open -a prometheus -i $INSTALLATION --no-browser                                                                                                                             (flux01-admin@flux01/default)
https://prometheus.g8s.test.gigantic.io:443```

**Comment by yulianedyalkova** [comment]
@pipo02mix what version of `opsctl` are you using? Trying it with `master`/`v2.8.0` works for me:
```
→ opsctl open -a happa -i galaxy --no-browser
https://happa.galaxy.test.gigantic.io

→ opsctl open -a happa -i anubis --no-browser
https://happa.g8s.he.k8s.emea.adsint.biz:443/admin-login
```

**Comment by yulianedyalkova** [comment]
@pipo02mix what version of `opsctl` are you using? Trying it with `master`/`v2.8.0` works for me.

**Comment by pipo02mix** [comment]
yeah I released `2.8.0` and now it works
